<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>textureDialog</class>
 <widget class="QDialog" name="textureDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>398</width>
    <height>228</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Texture properties</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>381</width>
     <height>40</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="nameBox">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>20</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>67</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>46</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Kind</string>
   </property>
  </widget>
  <widget class="QComboBox" name="kindBox">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>50</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Solid color</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Mask texture</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Image texture</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Normap map</string>
    </property>
   </item>
  </widget>
  <widget class="QGroupBox" name="imageGroupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>371</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string>Image texture</string>
   </property>
   <widget class="QLineEdit" name="textureFileName">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>30</y>
      <width>281</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QToolButton" name="selectImageButton">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>30</y>
      <width>25</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>46</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>File:</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="maskGroupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>271</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>Mask texture</string>
   </property>
   <widget class="QComboBox" name="whiteMaskBox">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>20</y>
      <width>181</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="blackMaskBox">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>50</y>
      <width>181</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="textureMaskBox">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>80</y>
      <width>181</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>46</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>White</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>46</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Black</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>46</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Texture</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="solidColorGroupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>201</width>
     <height>110</height>
    </rect>
   </property>
   <property name="title">
    <string>Solid color</string>
   </property>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>46</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>R</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>46</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>G</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>46</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>B</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="blueColorBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <double>1.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="greenColorBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <double>1.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="redColorBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <double>1.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="normalMapGroupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>371</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string>Normal map</string>
   </property>
   <widget class="QLineEdit" name="mapTextureFileName">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>20</y>
      <width>281</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QToolButton" name="mapSelectImageButton">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>20</y>
      <width>25</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>46</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>File:</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="meanBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Mean</string>
    </property>
   </widget>
  </widget>
  <widget class="QDoubleSpinBox" name="tileUBox">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>20</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="singleStep">
    <double>0.100000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="tileVBox">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>50</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="singleStep">
    <double>0.100000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>20</y>
     <width>46</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Tile U</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>50</y>
     <width>46</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Tile V</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>textureDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>textureDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
